src/Language/PureScript/CST/Flatten.hs

```diff
@@ -151,6 +151,7 @@ flattenExpr = \case
   ExprRecordAccessor _ a -> flattenRecordAccessor a
   ExprRecordUpdate _ a b -> flattenExpr a <> flattenWrapped (flattenSeparated flattenRecordUpdate) b
   ExprApp _ a b -> flattenExpr a <> flattenExpr b
+  ExprVisibleTypeApp  _ a _ b -> flattenExpr a <> flattenType b
```

Comment:
It's been a while since I've looked at this code, so sorry if this is incorrect. But shouldn't the `@` token be included here?

Reply:
Yup